cosmo_tester/test_suites/image_based_tests/inplace_upgrade_test.py

```diff
@@ -31,47 +34,54 @@
 
 
 @pytest.fixture(scope='module', params=VERSIONS)
-def image_based_manager(request, cfy, ssh_key, module_tmpdir, attributes,
-                        logger):
-    instances = [get_image(request.param)]
-    hosts = TestHosts(cfy, ssh_key, module_tmpdir, attributes, logger,
-                      instances=instances, request=request)
+def manager_and_vm(request, cfy, ssh_key, module_tmpdir, attributes,
+                   logger):
+    hosts = Hosts(cfy, ssh_key, module_tmpdir, attributes, logger, 2)
+    hosts.instances[0] = get_image(request.param)
+    manager, vm = hosts.instances
+
+    manager.upload_files = False
+    manager.restservice_expected = True
+
+    vm.upload_files = False
+    vm._image_name = ATTRIBUTES['centos_7_image_name']
+    vm._linux_username = ATTRIBUTES['centos_7_username']
```

Comment:
`centos_7_image_name` and `centos_7_username` can be moved to a constant

Reply:
They're pulled from the configuration, but I am thinking about ways we might be able to make the whole TestHosts and VM stuff better.